tests/integration/components/security/download-app-test.js

```diff
@@ -0,0 +1,90 @@
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupIntl } from 'ember-intl/test-support';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { fillIn, click } from '@ember/test-helpers';
+import Service from '@ember/service';
+
+class NotificationsStub extends Service {
+  errorMsg = null;
+  successMsg = null;
+
+  error(msg) {
+    this.errorMsg = msg;
+  }
+  success(msg) {
+    this.successMsg = msg;
+  }
+}
+
+class WindowStub extends Service {
+  url = null;
+  target = null;
+
+  open(url, target) {
+    this.url = url;
+    this.target = target;
+  }
+}
+
+module('Integration | Component | security/download-app', function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks);
+
+  hooks.beforeEach(async function () {
+    this.owner.register('service:notifications', NotificationsStub);
+    this.owner.register('service:browser/window', WindowStub);
```

Comment:
you can stub window using the library test support `setupBrowserFakes(hooks, { window: true });`